src/App.tsx

```diff
@@ -1,61 +1,131 @@
+import { useState, ChangeEvent } from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+function getUser(userId: number) {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  // if there is no user with a given userId
+  return foundUser || null;
+}
+
+export const tasks = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
 
 export const App = () => {
+  const [taskUser, setTaskUser] = useState('0');
+  const [task, setTask] = useState('');
+  const [errorTitle, setTitleError] = useState(false);
+  const [errorUser, setUserError] = useState(false);
+  const [todos, setTasks] = useState(tasks);
```

Comment:
tasks here a based on some API response that was additionally parsed and enhanced with user property. You can't use them as a default value. In theory, it's something async that will be added later. 
Default value should be just an empty array.